database/AddArbeitszeit.java

```diff
@@ -0,0 +1,192 @@
+package database;
+
+import java.sql.*;
+
+/**
+ * Ein Programm zum Anlegen und Ändern von Zeitbuchungen.
+ *
+ * @author Simon Valiente
+ * @version 1.2
+ */
+
+public class AddArbeitszeit{
+
+  private Connection connection; // Erstellt ein Objekt der Klasse Connection
+  private String workDate;
+  private int MA_ID;
+
+  /**
+   * Konstruktor, ruft die Methode zum Aufbau der Verbindung zur DB auf
+   * und legt bereits fest, was beim Schließen des Programms passiert
+   *
+   * @param workDate ist das Datum des Arbeitstages im Format yyyy-mm-dd
+   * @param MA_ID ist die Personalnummer des/der Mitarbeiter:in
+   *
+   * @see createConnection()
+   */
+  public AddArbeitszeit(String workDate, int MA_ID){
+    this.workDate = workDate;
+    this.MA_ID = MA_ID;
+    createConnection();
+    Thread shutDownHook = new Thread() {
+      public void run() {
+        System.out.println("Running shutdown hook");
+        if(connection == null)
+          System.out.println("Connection to database already closed");
+        try {
+          if (connection != null && !connection.isClosed()){
+            connection.close();
+            if (connection.isClosed())
+              System.out.println("Connection to database closed");
+          }
+        } catch (SQLException e){
+          System.err.println("Shutdown hook couldn't close database connection.");
+        }
+      }
+    };
+    Runtime.getRuntime().addShutdownHook(shutDownHook);
+  }
+
+  /**
+   * Stellt Verbindung zum MySQL-Treiber her.
+   * Misslingt die Verbindung, ist womöglich der CLASSPATH falsch gesetzt
+   */
+
+  static {
+    try{
+      Class<?> c = Class.forName("com.mysql.cj.jdbc.Driver");
+      if (c != null){
+        System.out.println("JDBC-Treiber geladen");
+      }
+    } catch (ClassNotFoundException e){
+      System.err.println("Fehler beim Laden des JDBC-Treibers");
+      System.exit(1);
+    }
+  }
+
+  /**
+   * Stellt die Verbindung zur Datenbank her
+   */
+
+  private void createConnection(){
+    String url = "jdbc:mysql://localhost/zeiterfassung";
+    String user = "root";
+    String pass = "";
+    try{
+      System.out.println("Creating DBConnection");
+      connection = DriverManager.getConnection(url, user, pass);
+    } catch (SQLException e){
+      System.err.println("Couldn't create DBConnection");
+      System.exit(1);
+    }
+  }
+
+  /**
+   * Legt in der Datenbank einen Eintrag für die Arbeitszeit an
+   * Ein SQL-Befehl wird erstellt und an die Datenbank übergeben, wo dieser ausgeführt wird.
+   *
+   * @param beginTime ist der Beginn der Arbeitszeit im Format hh:mm:ss
+   * @param endTime ist das Ende der Arbeitszeit im Format hh:mm:ss
+   * @param totalBreak ist die Gesamtzeit der Pausen an dem Tag im Format hh:mm:ss
+   * @param overtime ist die Anzahl der Überstunden, die an dem Tag geleistet wurden im Format hh:mm:ss
```

Comment:
refactor to flexiTime or explain overtime means plus or minus hours

Reply:
only adjust comment with plus & minus based on further class wording :)

Reply:
So... should it's name stay overtime or should I refactor it to flexiTime?